scripts/repo/winesapos-build-repo.sh

```diff
@@ -78,6 +78,9 @@ makepkg_fn inputplumber-bin install
 makepkg_fn powerstation-bin install
 makepkg_fn opengamepadui-bin install
 makepkg_fn opengamepadui-session-git
+# Virtual Machine Containers
+makepkg_fn podman-desktop
```

Comment:
This repository build script is only for packages that are (1) from the AUR and (2) do not already have a Chaotic AUR package. We help fill in the gap by providing them in our winesapOS repository.

`podman-desktop` is in the Chaotic AUR so it is not needed here but `docker-desktop` is missing.

Besides that, I should have better communicated this feature request. I'm sorry for that! We already install both `docker` and `podman` (the CLI tools). I was simply looking to update the `CONTRIBUTING.md` documentation with steps on how to use `docker` and `podman` for building winesapOS.

We could still potentionally ship these `*-destop` GUI tools. The install size of `podman-desktop` is almost 350 MiB which is a bit large. I could offer it as an option to install during the first-time setup, at least. That way it doesn't take up precious space on the pre-built image.

Reply:
I Will remove the GUI `*-desktop` packages and use the `CLI` packages